### PR TITLE
Merge GSA 9 into 20.08

### PR DESCRIPTION
**Pull request by swaterkamp**
**Checklist**:

- [N/A] Tests
- [N/A] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/gmp/utils/entitytype.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ const TYPES = {
   report_format: 'reportformat',
   tls_certificate: 'tlscertificate',
   vuln: 'vulnerability',
+  allinfo: 'secinfo',
```
Comment (sarahd93) [inline gsa/src/gmp/utils/entitytype.js:58]:
```suggestion
```

File: gsa/src/web/entities/container.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ import {
   apiType,
   typeName,
   pluralizeType,
+  normalizeType,
```
Comment (sarahd93) [inline gsa/src/web/entities/container.js:38]:
```suggestion
```

File: gsa/src/web/entities/container.js [hunk 3]
```diff
@@ -84,7 +85,7 @@ class EntitiesContainer extends React.Component {
 
     const {gmpname, gmp, notify} = this.props;
 
-    const entitiesCommandName = pluralizeType(gmpname);
+    const entitiesCommandName = pluralizeType(normalizeType(gmpname));
```
Comment (sarahd93) [inline gsa/src/web/entities/container.js:88]:
```suggestion
    const entitiesCommandName = pluralizeType(gmpname);
```

File: gsa/src/gmp/utils/__tests__/entitytype.js [hunk 4]
```diff
@@ -88,6 +88,7 @@ describe('normalizeType function tests', () => {
     expect(normalizeType('report_format')).toEqual('reportformat');
     expect(normalizeType('config')).toEqual('scanconfig');
     expect(normalizeType('vuln')).toEqual('vulnerability');
+    expect(normalizeType('allinfo')).toEqual('secinfo');
```
Comment (sarahd93) [inline gsa/src/gmp/utils/__tests__/entitytype.js:91]:
```suggestion
```

**Comment by swaterkamp** [comment]
Good catch, thank you!

**Pull request merged**